### PR TITLE
Extend get twins endpoint for pagination parameters

### PR DESCRIPTION
**Pull request by LuLeRoemer**

### CODE REVIEW

**Review by JaneMcBrain — APPROVED** [review]

File: semantics/services/digital-twin-registry-openapi.yaml [hunk 1]
```diff
@@ -31,6 +31,24 @@ paths:
             maxLength: 200
           name: value
           in: query
+        - in: query
+          name: pageSize
+          required: true
```
Comment (drcgjung) [inline semantics/services/digital-twin-registry-openapi.yaml:36]: Having a parameter "required" and providing a default value is not consistent?

File: semantics/services/digital-twin-registry-openapi.yaml [hunk 2]
```diff
@@ -31,6 +31,24 @@ paths:
...
+            default: 10
+            type: integer
+            enum:
+              - 10
+              - 50
+              - 100
+            description: Size of the pages that the results should be partitioned in
+        - in: query
+          name: page
+          required: true
```
Comment (drcgjung) [inline semantics/services/digital-twin-registry-openapi.yaml:47]: see above

**Review by drcgjung — APPROVED** [review]
see my comments, required parameters are a potential breaking change.

**Review by drcgjung — COMMENTED** [review]
Very good, thx.

**Pull request merged**